Cargo.toml

```diff
@@ -43,6 +43,11 @@ name = "cargo-upgrade"
 path = "src/bin/upgrade/main.rs"
 required-features = ["upgrade"]
 
+[[bin]]
+name = "cargo-set-version"
```

Comment:
> what do you think of using cargo bump instead of cargo set-version? E.g. `cargo bump 1.0.0` or `cargo bump minor`.

I did bring up that the current name is what drove me to not overload the version field with relative bumps but to push that out to `--bump`.  The side benefit is error reporting is more complicated when mixing fields.

Personally, I have no real preference within the scope of `cargo-edit`.  I do wonder what would be the more appropriate name if we upstream this into `cargo`.  I would suspect a more explicit name (ie has `version` in it).

I thought I'd look around for other tools
- [yarn calls it `version`](https://yarnpkg.com/cli/version)
- Same with [lerna](https://github.com/lerna/lerna/tree/main/commands/version#readme)
- The canonical python tool is [bumpversion](https://pypi.org/project/bumpversion/)
- [Some other misc tools](https://github.com/c4urself/bump2version/blob/master/RELATED.md)
  - While CalVer isn't good for libs, wonder if we should support it for bins.  Maybe enough of a corner case to leave it to people to give the version specifically.